store/tvShows.ts

```diff
@@ -5,42 +5,42 @@ import {
   BaseItemDtoQueryResult,
   ItemFields
 } from '@jellyfin/client-axios';
-import { AppState } from './index';
 
-interface TvShowItem {
+export interface TvShowItem {
   /**
    * seasons: Stores an array of all seasons
    */
   seasons: BaseItemDto[];
   /**
    * seasonEpisodes: Stores an array for each season containing all the season episodes
    */
-  seasonEpisodes: BaseItemDto[][];
+  seasonEpisodes: { [key: string]: BaseItemDto[] };
 }
 
 export interface TvShowsState {
   [key: string]: TvShowItem;
 }
 
-const defaultState = (): TvShowsState => ({});
+export const defaultState = (): TvShowsState => ({});
 
 export const state = defaultState;
 
 type MutationPayload = {
   seasons: BaseItemDto[];
   seasonEpisodes: BaseItemDto[];
   itemId?: string;
+  seasonItemId?: string;
 };
 
-export const getters: GetterTree<TvShowsState, AppState> = {
+export const getters: GetterTree<TvShowsState, TvShowsState> = {
   getSeasons: (state) => ({ itemId }: { itemId: string }): BaseItemDto[] => {
```

Comment:
Why is it like this and not like this:
```suggestion
  getSeasons: (state) => ( itemId: string ): BaseItemDto[] => {
```

Reply:
My preference is to pass objects and then destructure them, it makes it easier if we have multiple items and only certain ones need to be defined, but I guess we don't need that here.

Reply:
@Camc314 That's fair in those cases, but yeah there I don't see the need :p thanks!